.env.example

```diff
@@ -5,10 +5,13 @@
 ## 1 = Mainnet
 CHAIN_ID=31337
 
-## Key used to encrypt secrets and sign ETH transactions
+## Key used to sign ETH transactions
```

Comment:
This may get combined with mnemonic -- keeping separate for now to keep footprint of this PR smaller.